test/errors/throw-error-with-getter-throw-traced.mjs

```diff
@@ -0,0 +1,27 @@
+import { spawnPromisified } from '../common/index.mjs';
```

Comment:
I know it wasn't added by this PR but why do we have a new `test/errors` category ?

Reply:
I don’t know, but I don’t know where else this should go? It seems like we’ve been testing all the errors stuff in `test/message`, but that’s very constrictive; whereas a test using this pattern allows full flexibility (test exit codes, etc.).

Reply:
I think it should go in `test/parallel`. There's documentation about the directories in https://github.com/nodejs/node/tree/main/test#test-directories